docs/en/mutations.md

```diff
@@ -52,17 +52,17 @@ You can also dispatch mutations using objects:
 
 ``` js
 store.dispatch({
-  type: 'INCREMENT',
-  payload: 10
+  type: 'INCREMENT_BY',
+  payload: [ 10 ]
```

Comment:
@CthulhuDen did you test your statement?
```javascript
console.log( ...[10] )  // prints '10' using node v5.10 and Chrome 51. 
```
@dddomin3 and @LinusBorg are right.
There's no special behavior for a list of one element.

I do think the example could be made clearer with a mutation that accepts multiple args.